src/api/structure/fixture.ts

```diff
@@ -21,6 +21,7 @@ interface FixtureInitOptions {
 
 export default class Fixture extends TestingUnit {
     public path: string;
+    public noConcurrency: boolean;
```

Comment:
```suggestion
    public disableCouncurrency: boolean;
```
to preserve consistency https://testcafe.io/documentation/402782/reference/test-api/fixture/disablepagecaching